cellpack/autopack/loaders/migrate_v1_to_v2.py

```diff
@@ -63,16 +63,34 @@ def get_representations(old_ingredient):
 
 
 def convert_rotation_range(old_ingredient):
-    range_min = (
-        old_ingredient["orientBiasRotRangeMin"]
-        if "orientBiasRotRangeMin" in old_ingredient
-        else -pi
-    )
-    range_max = (
-        old_ingredient["orientBiasRotRangeMax"]
-        if "orientBiasRotRangeMax" in old_ingredient
-        else pi
-    )
+    has_min = "orientBiasRotRangeMin" in old_ingredient
+    has_max = "orientBiasRotRangeMax" in old_ingredient
+
+    if has_min and has_max:
+        range_min = old_ingredient["orientBiasRotRangeMin"]
+        range_max = old_ingredient["orientBiasRotRangeMax"]
+        if range_min > range_max:
+            range_min, range_max = range_max, range_min
+    elif has_min and not has_max:
+        provided_min = old_ingredient["orientBiasRotRangeMin"]
+        if provided_min > pi:
+            range_min = -pi
+            range_max = provided_min
+        else:
+            range_min = provided_min
+            range_max = pi
+    elif has_max and not has_min:
+        provided_max = old_ingredient["orientBiasRotRangeMax"]
+        if provided_max < -pi:
+            range_min = provided_max
+            range_max = pi
+        else:
+            range_min = -pi
+            range_max = provided_max
+    else:
+        range_min = -pi
+        range_max = pi
+
```

Comment:
fixed a bug in the conversion function to prevent invalid ranges where `min>max` (e.g. `[6, pi]`)